### PR TITLE
do not merge 2

### PR DESCRIPTION
**Pull request by jmguzik**

### CODE REVIEW

**Comment by petr-muller** [comment]
/lgtm

This will not merge until someone triggers `e2e`

File: Makefile [hunk 1]
```diff
@@ -6,6 +6,8 @@
 #   test: Run all tests.
 #   clean: Clean up.
```
Comment (jmguzik) [inline Makefile:7]:
```suggestion
# this is my new suggestion
```

**Comment by jmguzik** [comment]
/uncc @petr-muller @hongkailiu 
Just to not bother you with test PR

**Pull request closed without merging**